src/ChipInput.js

```diff
@@ -210,7 +210,9 @@ class ChipInput extends React.Component {
 
   handleInputBlur = (event) => {
     if (!this.autoComplete.refs.menu) {
-      this.setState({ isFocused: false, inputValue: '' })
+      if (!this.props.persistInput) this.setState({ inputValue: '' });
```

Comment:
Eww... a semicolon! :smiley: Could you please use [Standard style](http://standardjs.com/rules.html) (no semicolons)?
I should add a linter and Travis integration, though...